src/DynamoRevit/DynamoRevit.csproj

```diff
@@ -48,6 +50,8 @@
     <Prefer32Bit>false</Prefer32Bit>
     <DocumentationFile>$(OutputPath)\DynamoRevitDS.XML</DocumentationFile>
     <DocumentationFile>$(OutputPath)\$(UICulture)\DynamoRevitDS.xml</DocumentationFile>
```

Comment:
As I can see from the summary, the xml files will not be generated. But it is not the case here.